### PR TITLE
Updated the texts in Examples

### PR DESCRIPTION
**Pull request by priyadarshu**
## Problem-

- All the examples had same **name** key value irrespective of different purposes of the tasks

## Action taken -

- Made changes in the name of the tasks under examples
- Corrected it with relevant  **name** key value to the comments of the task

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

-
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: docs/community.aws.iam_managed_policy_module.rst [hunk 1]
```diff
@@ -368,14 +368,14 @@ Examples
...
-    - name: Create IAM Managed Policy
+    - name: Update an IAM Managed Policy with new default version
       community.aws.iam_managed_policy:
         policy_name: "ManagedPolicy"
         policy: "{{ lookup('file', 'managed_policy_update.json') }}"
         state: present
 
     # Update a policy with a new non default version
-    - name: Create IAM Managed Policy
+    - name: Update an IAM Managed Policy with a non default version
```
Comment (Akasurde) [inline docs/community.aws.iam_managed_policy_module.rst:378]: @priyadarshu Thanks for the contribution. Documentation file like this RST is autogenerated. You may want to edit the actual code to reflect these changes [here](https://github.com/ansible-collections/community.aws/blob/3e30e37d7188e75a3ef19173e6a476742ce21e8e/plugins/modules/iam_managed_policy.py#L60). 

Let me know if you need any additional help.

**Review by Akasurde — CHANGES_REQUESTED** [review]

File: docs/community.aws.iam_managed_policy_module.rst (hunk 1 above)
Comment (priyadarshu) [reply docs/community.aws.iam_managed_policy_module.rst:378]: @Akasurde Thanks for letting me know, I will contribute on the actual code.

Comment (priyadarshu) [reply docs/community.aws.iam_managed_policy_module.rst:378]: @Akasurde I have raised a new PR, should I close this PR?

**Comment by Akasurde** [comment]
superseded by https://github.com/ansible-collections/community.aws/pull/918

**Pull request closed without merging**